### PR TITLE
Additional Material for 2017-2018 Lanetix Unionization Drive (Update 0074.md)

### PR DESCRIPTION
**Pull request by bwestergard**
Adding additional news sources and two primary sources. The first was the letter protesting management's retaliatory firing of a coworker who brought collective grievances to management, the second is the letter announcing our card majority, intention to file wit the board, and demand for voluntary recognition of the union.

### CODE REVIEW

**Comment by organizejs** [comment]
@nataliyaned

File: actions/0074.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
...ires-n73014474793/, https://medium.com/tech-workers-coalition/joint-statement-of-solidarity-with-unjustly-fired-lanetix-workers-354daaa4b306, https://www.bloomberg.com/news/articles/2018-02-26/coders-want-to-unionize-with-help-from-trump, http://www.sfweekly.com/news/s-f-tech-workers-protest-for-the-right-to-unionize/, https://nplusonemag.com/issue-31/politics/code-red/, http://www.spacedaily.com/afp/180427012948.386pka56.html, https://www.youtube.com/watch?v=ItihrPELgkA#t=1h26m27s, https://www.fastcompany.com/90209362/from-amazon-to-microsoft-tech-companies-faced-yet-another-ethical-reckoning-this-week, https://www.theguardian.com/commentisfree/2018/aug/09/silicon-valley-tech-workers-labor-activism, https://www.wired.com/story/labor-board-backs-startup-engineers-fired-unionizing/, https://www.fastcompany.com/90250388/the-advocate-bjorn-westergard, https://www.fastcompany.com/90244860/silicon-valleys-new-playbook-for-tech-worker-led-resistance, https://www.vice.com/en_us/article/598kyx/the-future-of-capitalism-is-san-francisco-on-steroids-v25n3, http://www.newsguild.org/mediaguild3/software-company-learns-a-tough-lesson-about-union-busting-fired-workers-win-big-back-pay-settlement/
```
Comment (nataliyaned) [inline actions/0074.md]: This link isn't working for me: https://www.bna.com/tech-startup-fires-n73014474793/

This link is repeated: https://gist.github.com/bwestergard/11f21ed3efa35284f653fce49178eaae

Comment (bwestergard) [reply actions/0074.md]: @nataliyaned Thanks for reviewing. I believe both problems have been fixed. Cheers!

**Comment by nataliyaned** [comment]
@organizejs Do you know why this check hasn't run yet?

**Comment by organizejs** [comment]
> @organizejs Do you know why this check hasn't run yet?

yeah, bc its merging in from an external branch, so github won't run it for security reasons (if it runs for any external fork, then people could make a PR with malicious code and our testing vm would run it)

If it looks good, we should merge with our admin rights, and our tests will run again once it is merged.

**Comment by bwestergard** [comment]
@organizejs It looks like my change was inadvertently reverted in this commit: https://github.com/organizejs/collective-actions-in-tech/commit/92b66452de7f49c49221cc10957aa95e588b0897#diff-f75d57a65c604ffc639d57334c0f4ccfL2

**Pull request merged**